packages/types/nft.ts

```diff
@@ -63,16 +70,18 @@ export type NftCardInfo = {
   imageUrl?: string
   // Metadata loaded from the token URI.
   metadata?: Record<string, any>
-  floorPrice?: {
-    amount: number
-    denom: string
+  highestOffer?: {
+    amount?: number | null
+    amountUsd?: number | null
+    denom?: string | null
```

Comment:
if there's no highest offer, can we fallback to collection floor price?

Reply:
Stargaze's API handles this already. It looks like they search through 4 sources

NAME_OFFER
TOKEN_OFFER
COLLECTION_OFFER
LIVE_AUCTION_OFFER

Reply:
oh amazing! that makes life so much easier for us